### ISSUE TITLE
Disable cite ID and url generation?

### ISSUE THREAD

**Issue by ghost** [inukshuk/jekyll-scholar#71]
Hi. I am hoping there is a way to disable the <a> tag generation on liquid cite tags. I put {% cite "source" %} into end notes (footnotes on bottom of page) using markdown. Anywhere you put {% cite "source" %} creates an ID link on the liquid tag that just links to itself, and I am not sure why this is or why it's needed. All I know is that is muddles up my end notes section with links that go nowhere. I would like to disable the <a> tag generation on the liquid tag.

**Comment by elotroalex** [comment]
Hi, @inukshuk, sorry to re-open this one. I do have a reason to want to change the href content in the generated link. Basically I want it to point to the ID in a separate bibliography page.

I was thinking of messing with:
```ruby
     def cite(keys)
        items = keys.map do |key|
          if bibliography.key?(key)
            entry = bibliography[key]
            entry = entry.convert(*bibtex_filters) unless bibtex_filters.empty?
          else
            return missing_reference
          end
        end

        link_to "##{[prefix, keys[0]].compact.join('-')}", render_citation(items)
      end
```
I would rather not, though, for elegance reasons. Not all my projects have separate bibliography pages, for example. And to be perfectly honest I'm not sure I understand the syntax inside the quotation marks after link_to. The double ## is throwing me off. Admittedly my Ruby is very rusty, and my CSL and Cite-Proc is all new. Perhaps I'm missing something obvious here. Any ideas?

**Comment by elotroalex** [comment]
Oh, and thanks for such an amazing plug-in! I'm incorporating it into a theme I'm building for literary editors: [Ed](https://github.com/elotroalex/ed). I'm in early alpha, but this is definitely making it into its very fabric.

**Comment by inukshuk** [comment]
There's nothing special about the double ## -- we just want to create a href like `#matz2008` because the respective reference on the same page will have id `matz2008`. If you want to link to another page you'd just leave of the # at the beginning.

I wouldn't mind making the link target configurable, but I haven't figured out a anything that would make it easier than just overriding the Ruby method. One thing I could think of, though, is we could add a separate helper to utilities to generate just the href target based on a given key. Then you'd just have to override that one method an not the entire cite method. What do you think?

Basically, something like:

    def link_target_for key
      "##{[prefix, key].compact.join('-')}"
    end

And then `link_to link_target_for(keys[0]), render_citation(items)`; this way you could then override `link_target_for` and return something else. I'd be happy to accept a PR along those lines.

**Comment by elotroalex** [comment]
Great idea! I added one of my own. Just sent a PL. Let me know what you think.
